### DISCUSSION TITLE
Token file generation

### DISCUSSION THREAD

**Discussion by japurcell** [material-components/material-web#5698] [Q&A] [answered]
What is being used to generate the token SASS files as in https://github.com/material-components/material-web/blob/main/tokens/v0_192/_md-comp-assist-chip.scss? The comment clearly states that this file was automatically generated. Just curious if there is another tool being used to help make design token management easier in my own system.

**Comment by asyncliz** [comment, marked as answer]
It is generated using an internal tool that reads token data from the same source that serves tokens on [material.io](https://material.io/). Unfortunately it won't be made open source.
